com.github.TheThirdOne.rars.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
This file should be submitted upstream.

Reply:
The upstream source is [only distributed as a `.jar` file](https://github.com/TheThirdOne/rars#download), which uses the JVM to run (I don't believe freedesktop.org desktop entries are compatible with JAR files).